### PR TITLE
Fix test that started failing with v2.17.0 of qunit

### PR DESCRIPTION
**Pull request by kategengler**
Change test to look for output that asserts the same results, the
underlying behavior of qunit changed in version 2.17.0 and altered the
result.

### CODE REVIEW

**Comment by kategengler** [comment]
Submitted this issue to qunit https://github.com/qunitjs/qunit/issues/1651

**Comment by stefanpenner** [comment]
This unblocks an issue, but it seems like we may want to restore the original functionality if qunit fixes this issue. Given that, could you leave an inline commit + an issue so we eventually get back to this?

**Review by stefanpenner — APPROVED** [review]

File: tests/acceptance/brocfile-smoke-test-slow.js [hunk 1]
```diff
@@ -183,9 +183,8 @@ describe('Acceptance: brocfile-smoke-test', function () {
 
     let ember = path.join('.', 'node_modules', 'ember-cli', 'bin', 'ember');
 
-    let error = await expect(runCommand(ember, 'test', '--filter=jshint')).to.eventually.be.rejected;
-
-    expect(error.output.join('')).to.include('Error: No tests matched the filter "jshint"');
+    let response = await expect(runCommand(ember, 'test', '--filter=jshint')).to.eventually.be.ok;
+    expect(response.output.join('')).to.include('tests 0');
```
Comment (Krinkle) [inline tests/acceptance/brocfile-smoke-test-slow.js:187]: Hi :)

I'm trying to connect the dots between this and https://github.com/qunitjs/qunit/issues/1651.

Is the response output here by any chance based on the DOM's `Node.textContent` in some way? It appears not, but if not then I'm not exactly sure why this changed.

**Comment by stefanpenner** [comment]
'

**Comment by stefanpenner** [comment]
@kategengler alternatively, should we pin to an older QUnit for the time beind?

I'm open to either approach, just curious as to your thoughts

File: tests/acceptance/brocfile-smoke-test-slow.js (hunk 1 above)
Comment (kategengler) [reply tests/acceptance/brocfile-smoke-test-slow.js:188]: `ember test` runs the qunit browser runner through `testem` and would reject when running and having a failing test.

**Comment by kategengler** [comment]
@stefanpenner I think we should just change the test -- the qunit is in the blueprints and it wouldn't prevent our users from upgrading it.

**Pull request merged**